recipes/pycomm3/LICENSE

```diff
@@ -0,0 +1,24 @@
+The MIT License (MIT)
```

Comment:
The PyPI tarball used here already includes this file, so don't include it manually.

Reply:
Great, wasn't aware of this. Removed it!